include/swift/SIL/SILBridging.h

```diff
@@ -146,6 +146,42 @@ enum class BridgedArgumentConvention {
   Pack_Out
 };
 
+#ifdef USED_IN_CPP_SOURCE
```

Comment:
That's fine. But what we could also do is to ensure a 1-1 mapping of raw values of both enums and add static_asserts to enforce this. Like we do for other bridged enums, e.g. `PerformanceConstraints`

Reply:
Added. 

There actually isn't a one to one mapping b/w `BridgedArgumentConvention` and `swift::ParameterConvention` so the asserts do look a bit fragile. Let me know if you don't think I've done what you were expecting.

Reply:
Right. In C++ we have two separate convention enums: ParameterConvention, which is for the AST function type and ArgumentConvention, which is for lowered function types. Compared to ParameterConvention, the ArgumentConvention includes the out-conventions for indirect results.
On the swift side we only have ArgumentConvention for simplicity.

So ignore my suggestion and let's keep `getParameterConvention` and `getArgumentConvention` as you implemented it.